<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>312</width>
    <height>128</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>312</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Получить отчет</string>
  </property>
  <widget class="QPushButton" name="cancelStatsButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getFinaleStats">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Запрос</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Вы можете посмотреть&lt;br/&gt;статистику чеков, расход&lt;br/&gt;продуктов и денег, а также&lt;br/&gt;прибыль за определенный срок.&lt;br/&gt;Введите количество дней для рассчета:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="getDaysBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>365</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
